test/correctness/simd_op_check_hvx.cpp

```diff
@@ -169,9 +169,8 @@ class SimdOpCheckHVX : public SimdOpCheckTest {
         check("vavg(v*.h,v*.h):rnd", hvx_width / 2, i16((i32(i16_1) + i32(i16_2) + 1) / 2));
         check("vavg(v*.w,v*.w)", hvx_width / 4, i32((i64(i32_1) + i64(i32_2)) / 2));
         check("vavg(v*.w,v*.w):rnd", hvx_width / 4, i32((i64(i32_1) + i64(i32_2) + 1) / 2));
-        check("vnavg(v*.ub,v*.ub)", hvx_width / 1, i8_sat((i16(u8_1) - i16(u8_2)) / 2));
-        check("vnavg(v*.h,v*.h)", hvx_width / 2, i16_sat((i32(i16_1) - i32(i16_2)) / 2));
-        check("vnavg(v*.w,v*.w)", hvx_width / 4, i32_sat((i64(i32_1) - i64(i32_2)) / 2));
+        check("vnavg(v*.h,v*.h)", hvx_width / 2, i16((i32(i16_1) - i32(i16_2)) / 2));
```

Comment:
What happened to the u8 version?

Reply:
I put it back. It requires a special case, and I thought it was a marginal pattern.